Code/Framework/AzToolsFramework/Tests/ComponentModeTestFixture.h

```diff
@@ -10,14 +10,32 @@
 
 #include <AzToolsFramework/Application/ToolsApplication.h>
 #include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ToolsComponents/EditorDisabledCompositionBus.h>
+
 #include <AzCore/UnitTest/TestTypes.h>
 
 namespace UnitTest
 {
     class ComponentModeTestFixture
         : public ToolsApplicationFixture
+        , public AzToolsFramework::EditorDisabledCompositionRequestBus::Handler
     {
+    public:
+        // EditorDisabledCompositionRequestBus overrides ...
+        void GetDisabledComponents(AZStd::vector<AZ::Component*>& components) override;
+        void AddDisabledComponent(AZ::Component* componentToAdd) override;
+        void RemoveDisabledComponent(AZ::Component* componentToRemove) override;
+
+        void Connect(AZ::EntityId entityId);
+        void Disconnect();
+        // Adds disabled component to 
```

Comment:
Is this comment needed? It looks incomplete and most likely the function name `AddDisabledComponentToBus` is descriptive enough :p